docs/submitted_reports.md

```diff
@@ -0,0 +1,91 @@
+# Common report styles.
```

Comment:
could you link to this doc from the README, so we have a better chance of remembering it exists?